grouper/api/handlers.py

```diff
@@ -147,6 +155,34 @@ def get(self, name=None):
             })
 
 
+class BulkUsers(GraphHandler):
+    """API endpoint for bulk retrieval of user data.
+
+    This returns the same information as the Users and ServiceAccounts endpoints, but supports
+    multiple returning the data of multiple users to save on API call overhead.
+    """
+    def get(self):
+        names = self.get_argument("names", None)
```

Comment:
any reason not to use `get_arguments()` and let tornado do the parsing for us?

can we also call argument `username` instead (to be clear and to mostly follow convention)?